charts/management-portal/values.yaml

```diff
@@ -40,12 +40,12 @@ resources:
   # choice for the user. This also increases chances charts run on environments with little
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
+  limits:
+    cpu: 500m
```

Comment:
This will severely limit performance during startup. I'd request to remove this or set the limit to 2 cpu.

Reply:
Fixed in 3cce4da